manifests/init.pp

```diff
@@ -110,6 +110,8 @@
 #
 # $pulpcore_postgresql_ssl_root_ca::    Path to the root certificate authority to validate the certificate supplied by the PostgreSQL database server.
 #
+# $pulpcore_allowed_import_path::       allowed import paths for pulpcore settings.py. Used for Katello content view imports and repository sync with file protocol.
+# $pulpcore_allowed_export_path::       allowed export paths for pulpcore settings.py. Used for Katello content view exports.
```

Comment:
You can drop these now

Reply:
@ehelms thanks!  Updated and checks passing now :)